ComponentKit/Utilities/CKComponentAction.mm

```diff
@@ -76,8 +76,12 @@ void CKConfigureInvocationWithArguments(NSInvocation *invocation, NSInteger inde
   id responder = ([target respondsToSelector:@selector(targetForAction:withSender:)]
                   ? [target targetForAction:selector withSender:target]
                   : target);
-  CKCAssertNotNil(responder, @"Unhandled component action %@ following responder chain %@",
-                  NSStringFromSelector(selector), _CKComponentResponderChainDebugResponderChain(target));
+
```

Comment:
Why don't we just early-return if there's a nil selector here? There are real cases that are bugs when the target/responder are nil that we still want to assert on, right?